e2e/tests/components/marquee-wizard/marquee-wizard.e2e-spec.ts

```diff
@@ -271,6 +271,26 @@ describe('Marquee Wizard Tests', () => {
         expect(await page.resetButton.getText()).toBe('RESET STEP 1');
     });
 
+    it('should allow tabbing to the first step when using the keyboard', async () => {
+
+        // tab to first item on list
+        await browser.actions().sendKeys(Key.TAB).perform();
+
+        expect(await imageCompare('marquee-wizard-step-tab')).toEqual(0);
```

Comment:
Instead of a screenshot, it would be more efficient just to check which element has focus (`activeElement`).

Screenshots are intended to protect against visual regressions.